### PR TITLE
Remove irrelevant boolean check

### PR DESCRIPTION
**Pull request by yaman-jain**

### CODE REVIEW

File: zlibWrapper/zstd_zlibwrapper.c [hunk 1]
```diff
@@ -220,7 +220,7 @@ ZEXTERN int ZEXPORT z_deflate OF((z_streamp strm, int flush))
         strm->next_out += dstCapacity;
         strm->total_out += dstCapacity;
         strm->avail_out -= dstCapacity;
-        if (flush == Z_FINISH && bytesLeft == 0) return Z_STREAM_END;
+        if (bytesLeft == 0) return Z_STREAM_END;
```
Comment (Cyan4973) [inline zlibWrapper/zstd_zlibwrapper.c]: cc @inikep .
There is a large update of zlib_wrapper ongoing.
I'm unsure if these change remains relevant after the update.

Comment (yaman-jain) [inline zlibWrapper/zstd_zlibwrapper.c]: @Cyan4973 Can you tell me on which branch this major update is going on

Comment (inikep) [inline zlibWrapper/zstd_zlibwrapper.c]: A good catch but the proposed change is not relevant now.
Currently I'm working on zlibWrapper at https://github.com/inikep/zstd/commits/zlibWrapper

**Comment by Cyan4973** [comment]
superceded by #384

**Pull request closed without merging**